src/main/java/core/basesyntax/figures/figurehandlers/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.figures.figurehandlers;
+
+import core.basesyntax.figures.Circle;
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.IsoscelesTrapezoid;
+import core.basesyntax.figures.Rectangle;
+import core.basesyntax.figures.RightTriangle;
+import core.basesyntax.figures.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURES_COUNT = 4; //from 0 to 4
+    private final Random random = new Random();
+
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int randomIndex = random.nextInt(FIGURES_COUNT) + 1;
```

Comment:
there is no sense to assign `FIGURES_COUNT` to `4` and then increment it here - why not assigning it to `5`?

Reply:
I can`t use `FIGURES_COUNT`  with value 5 properly in nextInt() method I think, because it will return values from 0 to 5